File: TestSamples/VBSamples/gr_Text.vb
Imports ABCObjects, GraphABC

Module gr_Text

Sub Main()
  Dim bt As TextABC
  Dim x As Integer
  
  x = 224
  bt = New TextABC(60, 110, 110, "Hello!", RGB(x,x,x))
  
  While x > 32
    Sleep(40)
    x = x - 32
    bt = CType(bt.Clone(), TextABC)
    bt.Color = RGB(x, x, x)
    bt.MoveOn(7, 7)
  End While
End Sub

End Module

File: ReleaseGenerators/Samples/Pas/Graphics/ABCObjects/gr_Text.pas
// Клонирование графических объектов
uses ABCObjects,GraphABC;

var bt: TextABC;
  
begin
  var x := 224;
  bt := new TextABC(60,110,110,'Hello!',RGB(x,x,x));
  while x>32 do
  begin
    Sleep(40);
    x -= 32;
    bt := bt.Clone;
    bt.Color := RGB(x,x,x);
    bt.MoveOn(7,7);
  end;
end.
